README.md

```diff
@@ -281,6 +281,7 @@ Violations of [Stacked Borrows] found that are likely bugs (but Stacked Borrows
 * [Aliasing mutable references in `sized-chunks`](https://github.com/bodil/sized-chunks/issues/8)
 * [`String::push_str` invalidating existing references into the string](https://github.com/rust-lang/rust/issues/70301)
 * [`ryu` using raw pointers outside their valid memory area](https://github.com/dtolnay/ryu/issues/24)
+* [`LazyArray::swap` method creating overlapping mutable references (ink! PR)](https://github.com/rust-lang/miri/issues/1364)
```

Comment:
I am confused by the `ink! PR` part here, is there a PR we could link to?

Reply:
The PR where this bug was caught is this:
https://github.com/paritytech/ink/pull/311
However, I am not sure it makes sense to link the PR since the PR is already linked via the linked miri issue.

Reply:
Ah yes, and that's not a PR for just a fix, it's a more general thing.

But then the link text is confusing, because it sounds like it links to an ink! PR.

Reply:
Shall I remove the "(ink! PR)" parentheses?

Reply:
That would be good I think -- but `ink!` should appear somehow. Maybe just replace `LazyArray::swap` by it? Like, 
> `ink!` creating overlapping mutable references

(or maybe without backticks)

Reply:
done